### DISCUSSION TITLE
open projects for selenuim testing

### DISCUSSION THREAD

**Discussion by ohoudfd** [community/community#132505] [Programming Help]
### Select Topic Area

Question

### Body

Hello am looking for an open project for selenium testing as hands-on experience, how can I find it ?

**Comment by thiunuwan** [comment]
You can find Selenium testing projects on GitHub.Search for “Selenium testing” in the search bar. You can use filters like “Repositories” to find projects or “Issues” to find open tasks. Look for issues with tags like “good first issue” or “help wanted” to get started.

- Search for Repositories: Use search terms like “Selenium testing” or “Selenium projects” on [GitHub](https://github.com/search).
- Explore Issues: Look for repositories with open issues tagged with “help wanted” or “good first issue” in the [Selenium](https://github.com/SeleniumHQ/selenium) and [automation](https://github.com/search?q=automation+label%3A%22good+first+issue%22) spaces.
- Example Searches: [Selenium Testing Projects](https://github.com/search?q=selenium+testing), [Open Source Testing Projects](https://github.com/search?q=testing+label%3A%22good+first+issue%22+label%3A%22help+wanted%22)

**Reply by ohoudfd** [reply]
Thank you so much it's really help
